### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by XhmikosR**
/CC @vladikoff 

AppVeyor doesn't seem to be enabled for this repo.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,5 +67,9 @@
     "difflet": "~0.2.3",
     "semver": "2.1.0",
     "shelljs": "~0.2.5"
-  }
+  },
+  "files": [
+    "lib",
+    "LICENSE-MIT"
```
Comment (sindresorhus) [inline package.json]: this is included by npm by default, so can be removed.

Comment (XhmikosR) [inline package.json]: I don't see that to be the case. README.md is included by default, not LICENSE.

Comment (sindresorhus) [inline package.json]: Should go without saying, but make sure you're on the latest npm version.

Comment (XhmikosR) [inline package.json]:
```
C:\Users\xmr\Desktop>node -v && npm -v
v0.12.0
2.5.1
```

Comment (sindresorhus) [inline package.json]:
```
~/dev/grunt XhmikosR-master*  
❯ npm -v                            
2.5.1
                                                                                                                                                              
~/dev/grunt XhmikosR-master*  
❯ tar -tf $(npm pack) | grep LICENSE
package/LICENSE-MIT
```

Comment (XhmikosR) [inline package.json]: I don't see it installed but anyway, I removed it.

**Pull request merged**